config.md

```diff
@@ -79,6 +79,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
 * **`options`** (array of strings, OPTIONAL) Mount options of the filesystem to be used.
   * Linux: supported options are listed in the [mount(8)][mount.8] man page. Note both [filesystem-independent][mount.8-filesystem-independent] and [filesystem-specific][mount.8-filesystem-specific] options are listed.
   * Solaris: corresponds to "options" of the fs resource in [zonecfg(1M)][zonecfg.1m].
+  * Windows: "ro" (readonly) is the only supported option for Windows.
```

Comment:
@wking Should we just change the wording to be the following?

```
Windows: runtimes MUST support ro, mounting the filesystem read-only when ro is given.
```

If `ro` is not a great option, is there a better replacement we should consider?